### PR TITLE
Ticket Splitting in v1.6

### PR DESCRIPTION
**Pull request by xaur**
Preview: https://github.com/decredcommunity/wiki/blob/ts16/wiki/ticket-splitting-v1.6.md

### CODE REVIEW

File: wiki/ticket-splitting-v1.6.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+This document covers the status of ticket splitting in v1.6 but it's not a tutorial - see [this page](https://decredvoting.com/ticketsplitting) for a good entry point with all instructions linked, and then you can connect with other users in [@dcrtktsplit](https://t.me/dcrtktsplit) Telegram or [#ticketsplitting](https://chat.decred.org/#/room/#ticketsplitting:decred.org) on Matrix.
+
+Note: Here we only talk about _VSP staking_, because users who can [solo](https://docs.decred.org/advanced/solo-proof-of-stake-voting/) stake likely don't need ticket splitting.
+
+## Can I downgrade from v1.6?
+
+No. If you have upgraded to v1.6, the only way to use ticket splitting again is to install v1.5, re-download the blockchain and re-create the wallet from seed.
+
+{ can multiple versions of Decrediton be installed on Windows and macOS via our recommended install paths? }
```
Comment (matheusd) [inline wiki/ticket-splitting-v1.6.md]: Not easily for lay users. All config files (including wallet db) are (OS-)user global, so only a single install of decrediton can run for a given user.

You could probably do this if you have multiple OS users and use a different version on each, but I haven't personally tested this config.

File: wiki/ticket-splitting-v1.6.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+Decrediton uses dcrd to handle the chain and dcrwallet to handle the wallet.
+
+Release notes for [dcrd v1.6](https://github.com/decred/decred-binaries/releases/tag/v1.6.0#dcrd-v160) warn: once blockchain data is migrated to new version there is no going back without re-downloading it with v1.5.
+
+dcrwallet v1.6 also does a one-way migration of the wallet file and older versions will not work with it. { correct? }
```
Comment (matheusd) [inline wiki/ticket-splitting-v1.6.md]: Correct. This is usually always true.

File: wiki/ticket-splitting-v1.6.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+## Why don't you guys fix it?
+
+There is strong indication that the demand for ticket splitting is not enought to justify pulling our scarce dev resources away from other important work.
+
+Over almost 3 years 2,679 split tickets were purchased at an average rate of 81/month, meaning that roughly 0.2% of tickets are being split. Largest individual month had 127 split ticket purchases (2020-Mar).
+
+In addition, we have this data [gathered](https://www.reddit.com/r/decred/comments/m547to/not_focusing_on_split_tickets_will_affect_us/gr0in3e/) from the blockchain:
+
+- split ticket purchases by month [chart](https://pasteboard.co/JSJHHdY.png)
+- split ticket purchases by the number of particpants [chart](https://pasteboard.co/JSJI0f0.png)
```
Comment (matheusd) [inline wiki/ticket-splitting-v1.6.md]: If at all possible, I'd recommend bringing in these pics locally to this repo and linking to it here (vs on pasteboard which could go away at any time).

Comment (xaur) [reply wiki/ticket-splitting-v1.6.md]: Done, thanks for pointing it out.

I try to keep text repos super light by not adding binaries, but turns out I can put them into an orphan branch and link there. It is much better than relying on external sources, but is still flexible for low traffic scenarios (e.g. fetch only text with `git clone --single-branch ...` and then fetch the heavy branch with binaries when feasible).

**Pull request closed without merging**